src/node/mini_miner.h

```diff
@@ -161,11 +168,10 @@ class MiniMiner
      * if it cannot be calculated. */
     std::optional<CAmount> CalculateTotalBumpFees(const CFeeRate& target_feerate);
 
-    /** Construct a new block template with all of the transactions and calculate the order in which
-     * they are selected. Returns the sequence number (lower = selected earlier) with which each
-     * transaction was selected, indexed by txid, or an empty map if it cannot be calculated.
+    /** Construct a new block template with all of the transactions and calculate both the order in
```

Comment:
nit: Consider removing 'both': it is prone to going stale with future change and it adds not much information